2014-09-19 Kai Pastor (OpenOrienteering)
- Never fail on missing CMake tests, as we remove all tests from source dir.

diff -up Qt5/qtbase/mkspecs/features/create_cmake.prf.orig Qt5/qtbase/mkspecs/features/create_cmake.prf
--- Qt5/qtbase/mkspecs/features/create_cmake.prf.orig	2014-09-19 06:24:44.980392179 +0200
+++ Qt5/qtbase/mkspecs/features/create_cmake.prf	2014-09-19 06:25:43.187516386 +0200
@@ -300,5 +300,6 @@ cmake_qt5_module_files.path = $$[QT_INST
 # with 'CONFIG -= create_cmake'
 !equals(CMAKE_MODULE_TESTS, -) {
     isEmpty(CMAKE_MODULE_TESTS): CMAKE_MODULE_TESTS = $$MODULE_BASE_INDIR/tests/auto/cmake
-    !exists($$CMAKE_MODULE_TESTS): error("Missing CMake tests.")
+    # Test dirs were removed in OpenOrienteering build.
+    # !exists($$CMAKE_MODULE_TESTS): error("Missing CMake tests.")
 }
